src/plugins/apimon/apimon.cpp

```diff
@@ -278,6 +278,7 @@ static void print_addresses(drakvuf_t drakvuf, apimon* plugin, const dll_view_t*
             json_object_object_add(j_rvas, target.target_name.c_str(), json_object_new_int(target.offset));
     }
 
+    // does this need to be converted to fmt?
```

Comment:
Looks like it

Reply:
Ah, I just checked. We don't have `drakvuf_trap_info* info` in the parameter. So, `fmt` can't be used ig

Reply:
In that case please remove the added comment

Reply:
Done